server/lobbyconnection.py

```diff
@@ -594,10 +594,10 @@ def decodeUniqueId(self, serialized_uniqueid):
                 SerialNumber = str(machine.get('SerialNumber', 0)).encode() # BIOS serial number
                 VolumeSerialNumber = str(machine.get('VolumeSerialNumber', 0)).encode() # https://www.raymond.cc/blog/changing-or-spoofing-hard-disk-hardware-serial-number-and-volume-id/
```

Comment:
:) I think that's a reminder that combining all features into a single hash is not a good idea. Feel free to remove this comment.